packages/resilience/README.md

```diff
@@ -0,0 +1,61 @@
+<!-- sap-cloud-sdk-logo -->
+<!-- This block is inserted by scripts/replace-common-readme.ts. Do not adjust it manually. -->
+<a href="https://sap.com/s4sdk"><img src="https://help.sap.com/doc/2324e9c3b28748a4ae2ad08166d77675/1.0/en-US/logo-with-js.svg" alt="SAP Cloud SDK for JavaScript Logo" height="122.92" width="226.773"/></a>
+<!-- sap-cloud-sdk-logo-stop -->
+
+# @sap-cloud-sdk/http-client
+
+This package contains implementations for the resilience middlewares like a timeout or circuit breaker middleware. 
+This concept of middlewares is totally generic, and you can use it for non-resilience use cases.
+
+## Installation
+
+```
+$ npm install @sap-cloud-sdk/resilience
+```
+
+## Usage
+
+Add a timeout middleware to a request.
+
+```
+executeHttpRequest(destination, {middleware:[timeout()], httpRequestOptions);
+```
+<!-- TODO v3 put better link when documentation is done -->
+For more detailed overview visit our [resilience documentation](https://sap.github.io/cloud-sdk/docs/js/).
+
+<!-- sap-cloud-sdk-common-readme -->
+<!-- This block is inserted by scripts/replace-common-readme.ts. Do not adjust it manually. -->
+## Support
+
+The recommended way to get in touch with us is to create an issue on [GitHub](https://github.com/SAP/cloud-sdk-js/issues).
+Select the issue category `Bug`, `Feature` or `Question` depending on the nature of your request.
```

Comment:
```suggestion
Select the issue category **Bug**, **Feature**, or **Question** depending on the nature of your request.
```

Reply:
Not sure if this is generated (or copy-pasted), but if yes, we should adjust it for all packages.